packages/oui-field/src/field.controller.js

```diff
@@ -13,11 +13,11 @@ const CONTROLS_SELECTORS = [
 ];
 
 const VALIDATION_PARAMETERS = {
-    min: ["min", "ng-min"],
-    max: ["max", "ng-max"],
-    minlength: ["minlength", "ng-minlength"],
-    maxlength: ["maxlength", "ng-maxlength"],
-    pattern: ["pattern", "ng-pattern"]
+    min: ["min", "ng-min", "ngMin"],
```

Comment:
We also could do something like `.map(attribute => _.camelCase(attribute))` in _component-utils.js_.

Reply:
to discuss after